packages/babel-parser/test/fixtures/experimental/nullish-coalescing-operator/no-paren-nullish-or/input.js

```diff
@@ -0,0 +1 @@
+e ?? f ?? g || h;
```

Comment:
Let's add an explicit test case for `a ?? b ?? c` not requiring parens.

Reply:
a test for this already exists here
https://github.com/babel/babel/blob/master/packages/babel-parser/test/fixtures/experimental/nullish-coalescing-operator/associativity/input.js#L1

Reply:
Perfect.